packages/mobx-state-tree/src/core/type/type.ts

```diff
@@ -45,35 +49,116 @@ export enum TypeFlags {
 
 export interface IType<C, S, T> {
     name: string
-    flags: TypeFlags
```

Comment:
`validate`, `is`, `describe` can remain public, they are very useful to just run validation

Reply:
done!